onnx/defs/schema.cc

```diff
@@ -861,13 +867,15 @@ OpName_Domain_Version_Schema_Map& OpSchemaRegistry::map() {
 
 #ifndef NDEBUG
       size_t dbg_registered_schema_count = GetRegisteredSchemaCount() - dbg_initial_schema_count;
-
-      ONNX_ASSERTM(
-          dbg_registered_schema_count == ONNX_DBG_GET_COUNT_IN_OPSETS(),
-          "%u schema were exposed from operator sets and automatically placed into the static registry.  "
-          "%u were expected based on calls to registration macros. Operator set functions may need to be updated.",
-          dbg_registered_schema_count,
-          ONNX_DBG_GET_COUNT_IN_OPSETS());
+      // Only checks if the schemas were entirely loaded
```

Comment:
nit change comment to : Only check if schemas for all opset versions are loaded

Reply:
How about "Check enabled only if schemas for all opset versions are loaded"?